src/utils/api-utils.js

```diff
@@ -1,5 +1,6 @@
 export const invItemURL = (locationName) =>
```

Comment:
can we rename this to invitemURL by stockRoom or something and the next const variable would not be conflicting with this name